client/webserver/site/src/js/forms.js

```diff
@@ -27,7 +33,7 @@ export class NewWalletForm {
     bind(form, fields.submitAdd, async () => {
       const pw = fields.nwAppPass.value || (this.pwCache ? this.pwCache.pw : '')
       if (!pw && !State.passwordIsCached()) {
-        fields.newWalletErr.textContent = 'app password cannot be empty'
+        fields.newWalletErr.textContent = window.locales.formatDetails(ID_NO_PASS_ERROR_MSG)
```

Comment:
Why is this one different now?  No longer says "app" password, but that's what this is referring to.